syntax = "proto3";

package api.user.event;

message ConnectionAccepted {
    uint64 initiator_user_id = 1;
    uint64 responder_user_id = 2;
}
